operational/azure/total_instance_hours/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## v2.0.1
+
+- Updated Microsoft URL referencing Azure Virtual Machine Families and ISF Ratios. Policy functionality is unchanged.
```

Comment:
Would reword this to describe the change in terms of user experience; basically, this prevented the policy from completing without error and has been fixed.